modules/charon-core/src/test/java/org/wso2/charon3/core/utils/ResourceManagerUtilTest.java

```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.charon3.core.utils;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.charon3.core.exceptions.BadRequestException;
+import org.wso2.charon3.core.exceptions.CharonException;
+import org.wso2.charon3.core.schema.AttributeSchema;
+import org.wso2.charon3.core.schema.SCIMAttributeSchema;
+import org.wso2.charon3.core.schema.SCIMResourceTypeSchema;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.wso2.charon3.core.schema.SCIMDefinitions.DataType.COMPLEX;
+import static org.wso2.charon3.core.schema.SCIMDefinitions.DataType.STRING;
+import static org.wso2.charon3.core.schema.SCIMDefinitions.Mutability.READ_WRITE;
+import static org.wso2.charon3.core.schema.SCIMDefinitions.Returned.DEFAULT;
+import static org.wso2.charon3.core.schema.SCIMDefinitions.Returned.NEVER;
+import static org.wso2.charon3.core.schema.SCIMDefinitions.Returned.REQUEST;
+import static org.wso2.charon3.core.schema.SCIMDefinitions.Uniqueness.NONE;
+
+/**
+ * Test class of ResourceManagerUtil.
+ */
+public class ResourceManagerUtilTest {
+
+    private SCIMResourceTypeSchema getResourceSchema() {
+        List<String> schemasList = new ArrayList<>();
```

Comment:
Let's add a newline after the method signature

Reply:
Fixed with b16f4bc341990ed28db680f0558364ff419804a4